models/business-unit/src/company/types.ts

```diff
@@ -0,0 +1,32 @@
+import type {
+  BusinessUnitKeyReference,
+  KeyReference,
+  Company,
+  CompanyDraft,
+} from '@commercetools/platform-sdk';
+import type { TBuilder } from '@commercetools-test-data/core';
+
+//BusinessUnitDraft
+export type TCompanyDraft = CompanyDraft;
+
+//BusinessUnitDraftGraphql is only scaffolding at this time
+export type TCompanyDraftGraphql = TCompanyDraft;
+
+//BusinessUnit
+export type TCompany = Company;
+export type TCompanyGraphql = Omit<TCompany, 'topLevelUnit' | 'parentUnit'> & {
+  ancestors: [];
+  inheritedStores: null;
+  __typename: 'BusinessUnit';
+  storesRef: KeyReference;
+  parentUnitRef: null;
+  parentUnit?: BusinessUnitKeyReference;
```

Comment:
In a `Company` the `parentUnit` is always `null` and the `topLevelUnit` is a self-reference

Reply:
Thank you @NickDevG! ❤️  I am confused about the distinction between the two in regard to their Graphql entities. I may tag this and work on it in another PR, but is there a good place to read on their differences other than the HTTP or Impex Docs?

Reply:
@tylermorrisford this is in the http API docs, but maybe is what youre looking for?
https://docs.commercetools.com/api/customers-overview#hierarchies-within-business-units

Reply:
@NickDevG We're making a new ticket to fix the Company type and to add the CompanyDraft and DivisionDraft transformers in a separate PR.